src/nrnpython/nrnpy_nrn.cpp

```diff
@@ -2120,7 +2120,7 @@ static int section_setattro(NPySecObj* self, PyObject* pyname, PyObject* value)
             PyErr_SetString(PyExc_ValueError, "bad value");
             return -1;
         }
-        *d = new_x;
+        d = new_x;
```

Comment:
What is this change?

Reply:
Ok, I investigated a bit and understood that `d` is a different type now. Instead of a plain assignment, which creates an intermediate container, we'r getting the pointer and assigning

Reply:
Actually, I noticed that `static_cast<double>(nb::float_(value))` would convert things to float, so rolling back to PyParse :(